### DISCUSSION TITLE
Is it possible to add Capella Space open data?

### DISCUSSION THREAD

**Discussion by flying318** [opengeos/opengeos.github.io#14] [Q&A]
Hi, 
I am very appreciated that you achieve a series of great projects for the geospatial community. I noticed that the offcial webpage of the Capella Space corporation recently released open sar data to [here](https://radiantearth.github.io/stac-browser/#/external/capella-open-data.s3.us-west-2.amazonaws.com/stac/catalog.json), which is a great geospatial resource. I wonder if it is possible to add a repository to search and download the specific event datas, like [maxar-open-data](https://github.com/opengeos/maxar-open-data.git). 😃 🌷

**Comment by giswqs** [comment]
Does it provide a STAC catalog that supports the search functionality?

**Reply by flying318** [reply]
Thanks for your reply. 😄 
Yep, I find the STAC catalogs [here](https://radiantearth.github.io/stac-browser/#/external/capella-open-data.s3.us-west-2.amazonaws.com/stac/catalog.json). `Capella Space` releases open data following the way below,
![image](https://github.com/user-attachments/assets/89463bbc-df43-47a2-9411-387752f501ba)
However, it is different from `maxar-open-data` with the way of specific disaster event. 😞

**Reply by giswqs** [reply]
It should be feasible to build an interactive web app for this. I will look into it when I have time.

**Reply by flying318** [reply]
Appreciated 😆
